web/menu.jspf

```diff
@@ -71,24 +71,153 @@ org.opensolaris.opengrok.web.Util"
             pMap.put(name, esc);
         }
 %>
+<script type="text/javascript" src="<%= request.getContextPath() %>/js/jquery-3.2.0.min.js"></script>
 <script type="text/javascript">/* <![CDATA[ */
-    document.domReady.push(function() { domReadyMenu(); });
+    document.xrefPath = '<%= request.getContextPath() + Prefix.XREF_P
+    %>';document.domReady.push(function() { domReadyMenu(); });
 /* ]]> */
+var initialWidth = null;
```

Comment:
I think this isn't really necessary to achieve the result. Combination of HTML markup duplication and hiding and showing based on css media queries looks cleaner (and perhaps faster than this).